e2e_tests/utils/utils.go

```diff
@@ -29,6 +29,7 @@ import (
 
 var (
 	yumInstallAgent = `
+sleep 10
```

Comment:
why this sleep?

Reply:
To attempt to deflake what may be a race condition with upgrading the agent on centos/rhel 7